### PR TITLE
F aws experiment - VPC Resource

### PR DESCRIPTION
**Pull request by rmenn**
This should have all the VPC resources defined except for Main routing table and Tags.

### CODE REVIEW

File: builtin/providers/aws/resource_aws_instance.go [hunk 1]
```diff
@@ -202,49 +203,57 @@ func resourceAwsInstanceCreate(d *schema.ResourceData, meta interface{}) error {
 		associatePublicIPAddress = v.(bool)
```
Comment (catsby) [inline builtin/providers/aws/resource_aws_instance.go]: Things like this need to be converted to the right `aws.Type`:

- https://github.com/awslabs/aws-sdk-go/blob/master/aws/types.go

So they end up like this: https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_autoscaling_group.go#L137-L139

File: builtin/providers/aws/resource_aws_vpc_test.go [hunk 2]
```diff
@@ -50,37 +51,6 @@ func TestAccVpc_dedicatedTenancy(t *testing.T) {
 	})
 }
 
-func TestAccVpc_tags(t *testing.T) {
```
Comment (catsby) [inline builtin/providers/aws/resource_aws_vpc_test.go]: Why remove this test?

**Comment by catsby** [comment]
Excellent start! Can you try making the acceptance tests pass?
You can run them like so:

    $ make testacc TEST=./builtin/providers/aws TESTARGS='-run=TestAccVpc'

Or individual ones by being more specific in the `-run`, like `TestAccVpcUpdate`. That should weed out a few of the issues like I mentioned above, regarding the `was` types.

Also, how does this related to pr #1040? There seems to be duplication, but it's lacking the connection setup. 

Thanks!

**Comment by rmenn** [comment]
I believe i didnt write the instance resource. But i can fix things. 

The tagging test was removed since i have not gotten time to write the tagging with the new library. Ill look over things and update.

I have closed out #1040, i believe i sent that while horribly sleepy.

**Comment by catsby** [comment]
Awesome, thanks. I'm looking forward to the revisions, thanks for the contributions!

**Comment by rmenn** [comment]
I am closing this, so i can rework and send it across again.

**Pull request closed without merging**